From 69174925594f46173e101187d92926777f33fb40 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Sun, 18 Dec 2022 21:09:00 -0600
Subject: [PATCH] Remove default pam.d folder

This conflicts with vendor pam.d files shipped in linux-pam
---
 factory/etc/pam.d/other       |  6 ------
 factory/etc/pam.d/system-auth | 23 -----------------------
 tmpfiles.d/etc.conf.in        |  3 ---
 3 files changed, 32 deletions(-)
 delete mode 100644 factory/etc/pam.d/other
 delete mode 100644 factory/etc/pam.d/system-auth

diff --git a/factory/etc/pam.d/other b/factory/etc/pam.d/other
deleted file mode 100644
index c1f729659c..0000000000
--- a/factory/etc/pam.d/other
+++ /dev/null
@@ -1,6 +0,0 @@
-# This file is part of systemd.
-
-auth     include system-auth
-account  include system-auth
-password include system-auth
-session  include system-auth
diff --git a/factory/etc/pam.d/system-auth b/factory/etc/pam.d/system-auth
deleted file mode 100644
index c2d6240930..0000000000
--- a/factory/etc/pam.d/system-auth
+++ /dev/null
@@ -1,23 +0,0 @@
-# This file is part of systemd.
-
-# You really want to adjust this to your local distribution. If you use this
-# unmodified you are not building systems safely and securely.
-
-auth      sufficient pam_unix.so
--auth     sufficient pam_systemd_home.so
-auth      required   pam_deny.so
-
-account   required   pam_nologin.so
--account  sufficient pam_systemd_home.so
-account   sufficient pam_unix.so
-account   required   pam_permit.so
-
--password sufficient pam_systemd_home.so
-password  sufficient pam_unix.so sha512 shadow try_first_pass use_authtok
-password  required   pam_deny.so
-
--session  optional   pam_keyinit.so revoke
--session  optional   pam_loginuid.so
--session  optional   pam_systemd_home.so
--session  optional   pam_systemd.so
-session   required   pam_unix.so
diff --git a/tmpfiles.d/etc.conf.in b/tmpfiles.d/etc.conf.in
index 2fdd498da5..b8f6287080 100644
--- a/tmpfiles.d/etc.conf.in
+++ b/tmpfiles.d/etc.conf.in
@@ -14,7 +14,4 @@ t /etc/mtab - - - - security.SMACK64=_
 {% endif %}
 C! /etc/locale.conf - - - -
 C! /etc/nsswitch.conf - - - -
-{% if HAVE_PAM %}
-C! /etc/pam.d - - - -
-{% endif %}
 C! /etc/issue - - - -
-- 
2.39.0

